Mathlib/Util/Delaborators.lean

```diff
@@ -122,9 +132,34 @@ def exists_delab : Delab := whenPPOption Lean.getPPNotation do
     | `(∃ $x:ident, $y:ident ≤ $z ∧ $body)
     | `(∃ ($x:ident : $_), $y:ident ≤ $z ∧ $body) =>
       if x == y then `(∃ $x:ident ≤ $z, $body) else pure stx
+    | `(∃ $x:ident, $y:ident ∉ $z ∧ $body)
+    | `(∃ ($x:ident : $_), $y:ident ∉ $z ∧ $body) => do
+      if x == y then `(∃ $x:ident ∉ $z, $body) else pure stx
+    | `(∃ $x:ident, $y:ident ⊆ $z ∧ $body)
+    | `(∃ ($x:ident : $_), $y:ident ⊆ $z ∧ $body) =>
+      if x == y then `(∃ $x:ident ⊆ $z, $body) else pure stx
+    | `(∃ $x:ident, $y:ident ⊂ $z ∧ $body)
+    | `(∃ ($x:ident : $_), $y:ident ⊂ $z ∧ $body) =>
+      if x == y then `(∃ $x:ident ⊂ $z, $body) else pure stx
+    | `(∃ $x:ident, $y:ident ⊇ $z ∧ $body)
+    | `(∃ ($x:ident : $_), $y:ident ⊇ $z ∧ $body) =>
+      if x == y then `(∃ $x:ident ⊇ $z, $body) else pure stx
+    | `(∃ $x:ident, $y:ident ⊃ $z ∧ $body)
+    | `(∃ ($x:ident : $_), $y:ident ⊃ $z ∧ $body) =>
+      if x == y then `(∃ $x:ident ⊃ $z, $body) else pure stx
     | _ => pure stx
-  -- Merging
   match stx with
   | `(∃ $group:bracketedExplicitBinders, ∃ $groups*, $body) => `(∃ $group $groups*, $body)
+  | `(∃ $b:binderIdent, ∃ $[$bs:binderIdent]*, $body) => `(∃ $b:binderIdent $[$bs]*, $body)
   | _ => pure stx
 end existential
+
+open Lean Lean.PrettyPrinter.Delaborator
+
+/-- Delaborator for `∉`. -/
+@[delab app.Not] def delab_not_in := whenPPOption Lean.getPPNotation do
+  let #[f] := (← SubExpr.getExpr).getAppArgs | failure
+  unless f.getAppFn matches .const `Membership.mem _ do failure
```

Comment:
There's `Expr.isAppOf` for this, but we should also check the arity since we can't be sure that expressions being pretty printed are well-formed.
```suggestion
  guard <| f.isAppOfArity ``Membership.mem 5
```